app/views/layouts/_google_analytics.html.erb

```diff
@@ -4,4 +4,5 @@
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments);}
   gtag('js', new Date());
+  gtag('config', "<%= code %>");
```

Comment:
is this `code` still available? seems that it's now removed

Reply:
it's in `effective_pages_helperl.rb#effective_pages_google_analytics`